### ISSUE TITLE
a11y: donor profile page missing h1 heading

### ISSUE THREAD

**Issue by ramiy** [WordImpress/Give#934]
In #819 I added admin headings, but it seems like I missed the "Donor" page.

**Comment by ramiy** [comment]
The admin heading review was in #820
